tests/DocBlock/TypeExpressionTest.php

```diff
@@ -739,7 +728,6 @@ private function parseTypeExpression(string $value, ?NamespaceAnalysis $namespac
         try {
             foreach ([false, true] as $pcreJit) {
                 ini_set('pcre.jit', $pcreJit ? '1' : '0');
```

Comment:
do we need to mess with JIT still?

Reply:
Yes, this is to test our regex /w and /wo JIT.